### ISSUE TITLE
Blender freeze using leaf render (with actual code)

### ISSUE THREAD

**Issue by lamogui** [wsmind/leaf#13]
The actual code freeze blender 2.79b (didn't try the other versions).
The freeze when selecting "rendered" visualisation mode with leaf engine selected

@wsmind I got some free time those days, do you have a debug workflow for blender plugins ?

**Comment by wsmind** [comment]
Hey lamogui :)

Maybe check if you're building the same target architecture (x86/x64) as your installed blender version?

**Comment by lamogui** [comment]
Yeah it's good, I couldn't compile the CUDA version but I deactivated the checkbox CUDA
